{{cookiecutter.slug}}/frontend.angular/proxy.conf.json

```diff
@@ -0,0 +1,6 @@
+{
+    "/api": {
```

Comment:
This shouldn't be just `/api`, but everything except for the static assets that aren't the index.html. See the Backbone frontend proxy for comparison.